### PR TITLE
Agent iOS test are finished. 

### PR DESCRIPTION
**Pull request by anastasia-tarasova**
Agent tests are finished. Temporary Cocoapods repo is used to test new zmq pod.

### CODE REVIEW

File: wrappers/ios/libsovrin-pod/libsovrin-demoTests/Case Tests/Agent/AgentHighCases.m [hunk 1]
```diff
@@ -300,7 +328,210 @@ - (void)testAgentListenWorksForAllDataInWalletPresent
...
+    
+    [TestUtils cleanupStorage];
+}
+
+// MARK: - Remove identity
+
+-(void)testAgentRemoveIdentityWorks
+{
+    [TestUtils cleanupStorage];
+    NSString *endpoint = @"127.0.0.1:9713";
```
Comment (jovfer) [inline wrappers/ios/libsovrin-pod/libsovrin-demoTests/Case Tests/Agent/AgentHighCases.m]: - [ ] change port to 98xx

File: wrappers/ios/libsovrin-pod/libsovrin-demoTests/Case Tests/Agent/AgentHighCases.m [hunk 2]
```diff
@@ -429,22 +671,29 @@ - (void)testAgentCloseConnectionWorksForOngoing
     
     // 3. listen
     NSString *endpoint = @"127.0.0.1:9705";
```
Comment (jovfer) [inline wrappers/ios/libsovrin-pod/libsovrin-demoTests/Case Tests/Agent/AgentHighCases.m]: - [ ] change port to 98xx

File: wrappers/ios/libsovrin-pod/libsovrin-demoTests/Case Tests/Agent/AgentMediumCases.m [hunk 3]
```diff
@@ -34,4 +29,310 @@ - (void)tearDown
     [super tearDown];
 }
 
+// MARK: - Add identity
+- (void)testAgentAddIdentityWorksForIncomingConnectionRequireLedgerRequestButPoolHandleIsInvalid
+{
+    [TestUtils cleanupStorage];
+    NSError *ret;
+    NSString *endpoint = @"127.0.0.1:9712";
```
Comment (jovfer) [inline wrappers/ios/libsovrin-pod/libsovrin-demoTests/Case Tests/Agent/AgentMediumCases.m]: - [ ] change port to 98xx

File: wrappers/ios/libsovrin-pod/libsovrin-demoTests/Case Tests/Agent/AgentMediumCases.m [hunk 4]
```diff
@@ -34,4 +29,310 @@ - (void)tearDown
...
+    NSString *pubKey;
+    ret = [[SignusUtils sharedInstance] createAndStoreMyDidWithWalletHandle:walletHandle
+                                                                       seed:nil
+                                                                   outMyDid:&did
+                                                                outMyVerkey:&verKey
+                                                                    outMyPk:&pubKey];
+    XCTAssertEqual(ret.code, Success, @"SignusUtils::createAndStoreMyDidWithWalletHandle() failed");
+    
+    // 3. listen
+    NSString *endpoint = @"127.0.0.1:9709";
```
Comment (jovfer) [inline wrappers/ios/libsovrin-pod/libsovrin-demoTests/Case Tests/Agent/AgentMediumCases.m]: - [ ] change port to 98xx

File: wrappers/ios/libsovrin-pod/libsovrin-demoTests/Case Tests/Pool/PoolHighCases.m [hunk 5]
```diff
@@ -323,7 +323,14 @@ - (void)testDeletePoolLedgerConfigWorksForOpened
     // 2. delete
     ret = [[PoolUtils sharedInstance] deletePoolWithName:poolName];
     XCTAssertEqual(ret.code, CommonInvalidState, @"PoolUtils::deletePoolWithName() returned wrong code!");
+    
     [TestUtils cleanupStorage];
+}
+
+- (void)testBunch
```
Comment (jovfer) [inline wrappers/ios/libsovrin-pod/libsovrin-demoTests/Case Tests/Pool/PoolHighCases.m]: - [ ] remove temporary bunch or make it more concrete and rename.

File: wrappers/ios/libsovrin-pod/libsovrin-demoTests/Test Utils/LedgerUtils.mm [hunk 6]
```diff
@@ -147,15 +147,15 @@ - (NSError *)buildAttribRequestWithSubmitterDid:(NSString *)submitterDid
     __block NSString *outJson = nil;
     NSError *ret;
     
-    NSString* hashStr = (hash) ? hash : @"";
-    NSString* rawStr = (raw) ? raw : @"";
-    NSString* encStr = (enc) ? enc : @"";
+    //NSString* hashStr = (hash) ? hash : @"";
```
Comment (jovfer) [inline wrappers/ios/libsovrin-pod/libsovrin-demoTests/Test Utils/LedgerUtils.mm]: - [ ] remove if this code actually not needed

File: wrappers/ios/libsovrin-pod/libsovrin-demoTests/Case Tests/Agent/AgentHighCases.m [hunk 7]
```diff
@@ -300,7 +328,210 @@ - (void)testAgentListenWorksForAllDataInWalletPresent
...
 }
 
-// MARK: - Agent seed
+// MARK: - Add identity
+
+- (void)testAgentAddIdentityWorks
+{
+    [TestUtils cleanupStorage];
+    NSError *ret;
+    NSString *endpoint = @"127.0.0.1:9711";
```
Comment (anastasia-tarasova) [inline wrappers/ios/libsovrin-pod/libsovrin-demoTests/Case Tests/Agent/AgentHighCases.m]: fixed

File: wrappers/ios/libsovrin-pod/libsovrin-demoTests/Case Tests/Agent/AgentHighCases.m [hunk 8]
```diff
@@ -300,7 +328,210 @@ - (void)testAgentListenWorksForAllDataInWalletPresent
...
+    XCTAssertEqual(ret.code, Success, @"SignusUtils::createAndStoreMyDidWithWalletHandle() failed for receiverDid 2");
+    
+    // 4. Add identity
+    ret = [[AgentUtils sharedInstance] addIdentityForListenerHandle:listenerHandle
+                                                         poolHandle:-1
+                                                       walletHandle:receiverWalletHandle
+                                                                did:receiverDid];
+    XCTAssertEqual(ret.code, Success, @"SignusUtils::createAndStoreMyDidWithWalletHandle() failed for receiverDid 2");
+    
+
```
Comment (jovfer) [inline wrappers/ios/libsovrin-pod/libsovrin-demoTests/Case Tests/Agent/AgentHighCases.m]: - [ ] remove call missed?

**Pull request merged**